File: sdkwork-app-sdk-go/types/wallet_transfer_form.go
package types

// 钱包转账请求
type WalletTransferForm struct {
	AccountType string `json:"accountType"`
	ToUserId string `json:"toUserId"`
	Amount float64 `json:"amount"`
	Points int `json:"points"`
	RequestNo string `json:"requestNo"`
	Remarks string `json:"remarks"`
}

File: sdkwork-app-sdk-typescript/src/types/wallet-transfer-form.ts
/** 钱包转账请求 */
export interface WalletTransferForm {
  /** 转账资产类型: CASH/POINTS */
  accountType: 'DEFAULT' | 'CASH' | 'POINTS' | 'TOKEN';
  /** 目标用户ID */
  toUserId: string;
  /** 现金转账金额(当accountType=CASH时必填) */
  amount?: number;
  /** 积分转账数量(当accountType=POINTS时必填) */
  points?: number;
  /** 幂等请求号(建议前端传UUID) */
  requestNo?: string;
  /** 备注 */
  remarks?: string;
}
